muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/notebook.scss

```diff
@@ -0,0 +1,219 @@
+    border: 2px solid $taskfield-border-color;
+    border-radius: 2px;
+    box-sizing: border-box;
+    margin: 4px 0 0;
+    max-width: 100%;
+    min-width: 100%;
+  }
+
+  /* stylelint-disable selector-class-pattern */
+  .cke_wysiwyg_div {
+    min-height: 150px;
+  }
+  /* stylelint-enable selector-class-pattern */
+
+  &.state-OPEN {
+    padding: 0 10px 10px;
+  }
+}
+
+.notebook__items {
+  display: flex;
+  flex-flow: column nowrap;
+  padding: 0 10px 10px;
+  position: relative;
+}
+
+.notebook__item {
+  border: 1px solid $color-default-separator-border;
+  border-radius: 3px;
+  box-shadow: 0 0 15px rgba(0 0 0 / 2%);
+  padding: 10px;
+  width: 100%;
+
+  &.state-DELETING {
+    border: 2px solid $color-fatal;
+    padding: 9px;
+  }
+}
+
+.notebook__item-delete {
+  align-items: center;
+  background: lighten($color-fatal, 40%);
+  display: flex;
+  flex-flow: column nowrap;
+  margin: 5px 0 10px;
+  padding: 10px;
+}
+
+.notebook__item-description {
+  font-weight: 400;
+  margin: 5px 0;
+}
+
+.notebook__item-buttonset {
+  align-items: center;
+  display: flex;
+  flex-flow: row nowrap;
+  margin: 5px 0;
+
+  .button {
+    margin: 0 5px;
+  }
+}
+
+.notebook__item-header {
+  align-items: flex-start;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: flex-start;
+}
+
+.notebook__item-title {
+  cursor: pointer;
+  font-size: 0.8125rem;
+  font-weight: 600;
+  margin: 0 0 10px;
+
+  @include breakpoint($breakpoint-pad) {
+    font-size: 0.9375rem;
+  }
+}
+
+.notebook__item-actions {
+  align-items: center;
+  display: flex;
+  justify-content: flex-end;
+  margin: 0 0 5px;
+  width: 100%;
+}
+
+.notebook__item-body {
+  &.rich-text {
+    font-size: 0.8125rem;
+
+    @include breakpoint($breakpoint-pad) {
+      font-size: 0.9375rem;
+      line-height: 1.5rem;
+    }
+  }
+
+  p {
+    margin: 10px 0;
+  }
+
+  p:first-child {
+    margin-top: 0;
+  }
+
+  p:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.note-item--drag-container {
+  align-items: center;
+  display: flex;
+  position: relative;
+}
+
+.note-item--drag-handle {
+  align-items: center;
+  background: $color-default;
+  border-radius: 100%;
+  color: $color-material-page-admin-panel-main-color;
+  cursor: grab;
+  display: flex;
+  font-size: 0.8125rem;
+  height: 1.37rem;
+  justify-content: center;
+  position: absolute;
+  right: 8px;
+  top: 3px;
+  width: 1.37rem;
+  z-index: 9999;
+}
+
+.draggable-element {
+  align-items: center;
+  display: flex;
+  flex-basis: auto;
+  flex-flow: row nowrap;
+  flex-grow: 0;
+  flex-shrink: 0;
+  margin: 0 0 10px;
+
+  &:last-child {
+    margin: 0;
+  }
+}
+
+.draggable-element__handle {
+  align-items: center;
+  background: $color-default-lighter-generic-background;
+  border-radius: 10px 0 0 10px;
+  display: flex;
+  height: 40px;
+  justify-content: center;
+  transition: width 0.3s;
+}
+
+.draggable-element__body {
+  flex-basis: 100%;
+  flex-grow: 1;
+  flex-shrink: 1;
+}
```

Comment:
These don't belong in this file.

Reply:
True, moving them to separate .scss file